.github/workflows/stale-routed-issues.yaml

```diff
@@ -0,0 +1,25 @@
+name: Stale issue action
+on:
+  schedule:
+    - cron: "* 8 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          any-of-labels: "frozen"
```

Comment:
In this case it only apply stale action to issues were triaged by the engineering teams.
Roadmap and other issues I don't think will be enter this workflow.